tests/feature/CountryTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+
+namespace Tychovbh\Tests\Mvc\feature;
+
+
+use Tychovbh\Mvc\Country;
+use Tychovbh\Mvc\Http\Resources\CountryResource;
+use Tychovbh\Tests\Mvc\TestCase;
+
+class CountryTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function itCanIndex()
```

Comment:
I see you added more country routes (store, show), so that means i'm missing other testcases here?
itCanStore
itCanShow

every route should always have a TestCase!